test/css/css_var_fallback_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "components/test_helper"
+
+class Primer::CssVariableTest < Minitest::Test
+  def test_css_rules_all_contain_fallbacks
+    # 1. Find compiled CSS file
+    css_file = Rails.root.join(*%w[.. app assets styles primer_view_components.css]).read
+
+    # 2. Load file and run regex on it which checks for any CSS var containing new color terms
+    regex = /var\(--(shadow|borderColor|bgColor|fgColor|iconColor)[^),]*\)\s*(,|;|\s)/
+    matches = css_file.scan(regex)
+
+    # 3. Use assert method (and friends) to verify it works
+    missing_vars = matches.map { |match| match[0] }
+    assert_equal 0, matches.length, "CSS variables are missing fallbacks: #{missing_vars.join(', ')}"
+  end
+end
```

Comment:
Here's what the output of this test look likes if it fails

<img width="830" alt="test failure" src="https://github.com/primer/view_components/assets/18661030/bc8e285d-90c0-4f19-b140-5564391e8d57">

I think since my regex is fuzzy, I can't get the full variable string. If anyone has feedback on how I can make this test failure more clear let me know!

Reply:
I added some logic in to point at the line and column so it's easier to find the failure.